judge/Makefile

```diff
@@ -28,8 +28,7 @@ runpipe: runpipe.cc $(LIBHEADERS) $(LIBSOURCES)
 install-judgehost:
 	$(INSTALL_PROG) -t $(DESTDIR)$(judgehost_libjudgedir) \
 		compile.sh build_executable.sh testcase_run.sh chroot-startstop.sh \
-		check_diff.sh evict version_check.sh
-	$(INSTALL_DATA) -t $(DESTDIR)$(judgehost_libjudgedir) \
```

Comment:
A line was dropped here.

Reply:
Yes, I read misread the INSTALL_DATA and assumed it was duplicated. Will remove the commit.